src/frontend/src/pages/CreateChangeRequestPage/CreateChangeRequestView.tsx

```diff
@@ -172,39 +231,33 @@ const CreateChangeRequestsView: React.FC<CreateChangeRequestViewProps> = ({
                 rows={4}
                 errorMessage={errors.what}
                 placeholder="What is the situation?"
-                sx={{ width: 300 }}
               />
             </FormControl>
           </Grid>
-          <Grid item xs={12} md={6}>
-            <FormControl>
-              <FormLabel>Why</FormLabel>
-              <Box>
-                {whys.map((_element, index) => (
-                  <Box display="flex" flexDirection="row" sx={{ mb: 1 }}>
-                    <NativeSelect {...register(`why.${index}.type`)}>
-                      {Object.values(ChangeRequestReason).map((type) => (
-                        <option key={type} value={type}>
-                          {type}
-                        </option>
-                      ))}
-                    </NativeSelect>
-                    <ReactHookTextField
-                      required
-                      multiline
-                      control={control}
-                      label="Explain"
-                      sx={{ flexGrow: 1, mx: 1, borderRadius: 2 }}
-                      {...register(`why.${index}.explain`)}
-                      errorMessage={errors.why?.[index]?.explain}
-                    />
-                    <IconButton type="button" onClick={() => removeWhy(index)}>
-                      <DeleteIcon />
-                    </IconButton>
-                  </Box>
-                ))}
-              </Box>
-            </FormControl>
+          <Grid item xs={12}>
+            <FormLabel>Why</FormLabel>
```

Comment:
Why'd you remove the FormControl

Reply:
theres an error about a form control having multiple input bases. I can try adding it back and seeing if it work